### ISSUE TITLE
Facebook log in button double circular progress bar

### ISSUE THREAD

**Issue by NeringaB** [firebase/FirebaseUI-Android#1052]
Hello, I would like to ask about Facebook log in button in Firebase UI. When the user clicks on the Facebook log in button, "Loading.." dialog appears as well as two circular progress bars. One where it should be to the left of the text "Loading..." and it is in the accent color of my app theme (red), another appears at the end of the word "Loading..." (half of the circular progress bar on "Loading...") and its color is Android default progress bar color. How could I fix this issue?

**Comment by samtstern** [comment]
@NeringaB can you show a screenshot of what you're seeing?  Having some trouble imagining it.

**Comment by NeringaB** [comment]
Okay, this is a screenshot, I have managed to catch a little bit of a second circular progress bar, it appears sometimes for a longer time, sometimes for a shorter, and disappears. I have also noticed that Facebook log in button works only on my own device, but on other devices (when a person downloads the app from Google Play through beta channel) it does not work. Though it seems that I did everything what is needed for a Facebook log in.

**Comment by NeringaB** [comment]
![screenshot_2017-12-15-11-16-28](https://user-images.githubusercontent.com/26273350/34036628-866c1058-e18e-11e7-8af4-352dfef7f6e2.png)

**Comment by samtstern** [comment]
@NeringaB huh ok I think the two bars are:

  - Red - that one is us.
  - Blue - the Facebook SDK does a transparent overlay with a blue progress bar during sign in.

Thanks for reporting this!  I'm not immediately sure what the best solution is but I think it will probably be part of #724 where we plan to remove all progress dialogs where possible.

**Comment by NeringaB** [comment]
Okay, thank you for your answer :)

**Comment by SUPERCILEX** [comment]
@samtstern I think we might as well close this issue in favor of #724, especially since Facebook is really fast when it comes to showing UI so I don't see the point of that progress bar at all. 👍

**Comment by samtstern** [comment]
Yes I guess we won't fix this unless as part of #724 so that can be where we track this, I will leave an extra comment there.
